custom_components/versatile_thermostat/strings.json

```diff
@@ -67,6 +68,7 @@
           "heater_entity2_id": "Optional 2nd Heater entity id. Leave empty if not used",
           "heater_entity3_id": "Optional 3rd Heater entity id. Leave empty if not used",
           "heater_entity4_id": "Optional 4th Heater entity id. Leave empty if not used",
+          "heater_keep_alive": "Optional heater switch state refresh interval. Leave empty if not required.",
```

Comment:
See comment https://github.com/jmcollin78/versatile_thermostat/pull/345#discussion_r1468544063.
Also, regarding the final period punctuation `’.’` at the end of _”Leave empty if not required.”_ , I have noticed that it is not consistent with the preceding descriptions that to not include the final period, but I included it because:
* You included the period in your suggested English and French revisions, and
* I think including the period is the right thing to do when the sentence is “full” rather than just a “label.” For example, _”Optional heater switch”_ is just a label, whereas _“This is an optional heater switch.”_ and _”Leave empty if not required.”_ are full sentences. (And if a sentence follows a label, a period is needed at the end of the label in order to separate it from the sentence.)